README.md

```diff
@@ -8,13 +8,17 @@
 
 ## Built With
 
-- HTML, CSS, JS
-- Bootstrap, SCSS, React, Redux
+- HTML, CSS, SCSS, JS
+-  React, Redux
+## Presentatin Link
 
+- [Video Demo](https://www.loom.com/share/e3fdfcaa2a2842a79baa3b318268fe62)
 
 ## Live Demo
 
-- [Live Demo Link]
+
+- [Live Demo Link](https://samiullahbahadur.github.io/Covid-tracker/)
```

Comment:
- [ ] I have to re-request to deploy your site and put the correctly deployed link on your README file as requested by the previous reviewer. Because it is still taking to the readme file. To solve this issue I think you can deploy from another branch, for example, the `covid `branch because it is up to date.

Reply:
Hi @elyor-doniyorov  
I did change the branch covid to dev and deploy

Reply:
But i think it's the github site issue